docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -174,40 +212,66 @@ Any configuration setting from a service's `configuration.yaml` file can be over
 
-    CONFIG : Clients:
-               core-data:
-                 Host: "localhost"
-    ENVVAR : CLIENTS_CORE_DATA_HOST=edgex-core-data    
-    ```    
+##### Private Configuration Overrides
 
-### SecretStore Overrides
+Each service has private configuration which is initially loaded from file. Private configuration overrides are only applied when the private configuration is load from file. These private configuration overrides must be set on the individual services since each service is loading its own private configuration file. 
+
+When using the Configuration Provider,  the values are pushed to the service's private section in the Configuration Provider. Once the private file and overrides have been used to seed the Configuration Provider, they are no longer used. The services will pull their private configuration directly from the Configuration Provider on future start-ups.
+
+When not using the Configuration Provider, the private overrides are applied when the configuration is load from this file and then used by the service. This occurs on every time a service starts.
+
+!!! important
+    Private configuration overrides are only applied to configuration settings that exist in the service's private configuration file.
+
+### SecretStore Configuration Overrides
 
 !!! edgey "EdgeX 3.0"
-    For EdgeX 3.0 the **SecretStore** configuration has been removed from each service's configuration files. It now has default values which can be overridden with environment variables.
+    For EdgeX 3.0 the **SecretStore** configuration has been removed from each service's configuration files and the Configuration Provider. It now has default values which can be overridden with [Configuration Overrides](#configuration-overrides).
 
-The environment variables overrides for **SecretStore** configuration remain the same as in 2.x releases. The following are SecretStore** fields that commonly need to be overridden.
+The environment variables overrides for **SecretStore** configuration follow the same rules as the regular configuration overrides. The following are the **SecretStore** fields that are commonly overridden.
 
 - SECRETSTORE_HOST
 - SECRETSTORE_RUNTIMETOKENPROVIDER_ENABLED
 - SECRETSTORE_RUNTIMETOKENPROVIDER_HOST
 
-The  complete list of **SecretStore** fields and defaults can be found in the file [here](https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/config/types.go) (search for SecretStoreInfo). The defaults for the remaining fields typically do not need to be overridden, but may be overridden if needed using that same naming scheme as above.
+!!! example - "Example SecretStore Configuration Override"
+    ```yaml
+    CONFIG : SecretStore.Host
+    ENVVAR : SECRETSTORE_HOST=edgex-vault
+    ```
```

Comment:
Same as before. Confusing combination of yaml and env var into a YAML object.

Reply:
For this one,  I took out the yaml tag since none of it is in YAML and expanded the labels